radio/src/gui/480x272/navigation.cpp

```diff
@@ -393,6 +405,25 @@ bool check(event_t event, uint8_t curr, const MenuHandlerFunc * menuTab, uint8_t
       break;
 
     case EVT_ROTARY_BREAK:
+      if (EDIT_SELECT_MENU == s_editMode)
+      {
+        if (menuTabModel == menuTab && (curr == MENU_MODEL_INPUTS
```

Comment:
I would prefer that those menus (INPUTS / MIXES / CURVES, etc.) set a flag which is used here